src/components/common/QuestionWrapper.vue

```diff
@@ -39,7 +39,7 @@ export default {
   counter-reset: header-counter;
 }
 
-h2::before {
+#question_container h2::before {
```

Comment:
How many question containers are rendered on a page? We should not have duplicate IDs, so maybe a class should be used here.

Reply:
Since the ID is declared in LimitedRestorationExtension.vue, and it is the parent of all the children question wrappers, I'm thinking this file is where the auto-numbering CSS belongs.

@jonathan-longe , what do you think?

Reply:
Modifying it was Jonathan's suggestions to make this work for restorations :)

Reply:
The `id="question_container"` has to exist in the parent in order for the `QuestionWrapper` to work.  I'd be okay changing it `id` to a `class` if there's going to be more than one set of prefix numbering displayed at a time.

Reply:
@severinbeauvais the point of adding the CSS to the 'QuestionWrapper` is so the logic and presentation is contained within the QuestionWrapper.  As a developer, you should be able to wrap the any common component and have it display the prefix numbering, as long as, you have a container labeled correctly using `id="questions_container"`.  If we move the CSS to parent, then we'll need to repeat the CSS everywhere it's needed which breaks the DRY principle.

Reply:
Understood, but having that CSS in there means a couple of icky things:
- the parent has to declare an ID on a wrapper
- the CSS has to be global

I'm not sure which is the best approach.

Reply:
> I'm not sure which is the best approach.

My SOP in this case is to not change anything, so I'm OK for leaving this for now.

Reply:
> Understood, but having that CSS in there means a couple of icky things:
> 
> * the parent has to declare an ID on a wrapper
> * the CSS has to be global

@severinbeauvais this is an important issue to address.  In my view, CSS _is_ designed to be applied globally.  Yes, it's true, CSS can be put into the `<style>` section of a Vue component and this is an okay approach if the style should only apply to the component, but if we restrict ourselves to only applying styles per component, we won't enjoy the full benefits of CSS.  

But, I hear your concern.  Rather than placing the CSS in the QuestionsWrapper component with "scoped" removed from `<style scoped>`, it would be cleaner if we moved the CSS to a file in `/src/assets/sytles/`?  That way we don't end up having globally applicable CSS buried inside a component.

Reply:
I think there are benefits to having scoped CSS as well as global CSS.

In this case, yes, this CSS is "more global" than the component in question, so moving it to the base CSS file should be considered. Do we know for sure that `question_wrapper` is used on only 1 page at a time? If not then this might not work as pages may use the same number value...?

Reply:
>  Do we know for sure that question_wrapper is used on only 1 page at a time? If not then this might not work as pages may use the same number value...?

Yes, from what I've seen, that's true.  If it turns out not to be true, then perhaps we could use a class instead of an id.

Reply:
OK, if it works for now then let's leave it as as ID for now.

> Rather than placing the CSS in the QuestionsWrapper component with "scoped" removed from <style scoped>, it would be cleaner if we moved the CSS to a file in /src/assets/sytles/? That way we don't end up having globally applicable CSS buried inside a component.

@ketaki-deodhar In your next PR, could you please the CSS to `base.scss` or `layout.scss`?

Thanks. I'll resolve this conversation now.